src/components/failedTaskBanner/index.tsx

```diff
@@ -63,26 +53,27 @@ const countFailedProcesses = (processes: ProcessWithStatus[]): CountFailedProces
 
 export default function FailedTaskBanner() {
     const { runningProcesses, completedProcessIds, useFallback } = useContext(RunningProcessesContext);
-    const [data, , fetchData] = useFailedTaskFetcher<FailedProcess>("processes/");
-    const [failedProcesses, setFailedProcesses] = useState<FailedProcess[]>([]);
-    const [failedTasks, setFailedTasks] = useState(countFailedProcesses([]));
+    const { apiClient } = useContext(ApplicationContext);
+    const [summary, setSummary] = useState<ProcessStatusCounts>();
+    const [failedTasks, setFailedTasks] = useState<CountFailedTasks>(countFailedTasks(summary));
     const history = useHistory();
     const handleOnClick = useCallback(() => history.push("/tasks"), [history]);
 
-    useHttpIntervalFallback(useFallback, () => fetchData(0, 10, [], filterFailedTasks));
-
-    useEffect(() => {
-        const newList = [...data, ...runningProcesses.filter((p) => failedProcessStatuses.includes(p.last_status))];
-        setFailedProcesses(newList.filter((p) => !completedProcessIds.includes(p.pid)));
-    }, [data, runningProcesses, completedProcessIds]);
+    useHttpIntervalFallback(useFallback, () =>
+        apiClient.processStatusCounts().then((res) => {
+            setSummary(res);
+        })
+    );
 
     useEffect(() => {
-        setFailedTasks(countFailedProcesses(failedProcesses));
-    }, [failedProcesses]);
+        apiClient.processStatusCounts().then((res) => {
```

Comment:
Create function for this, since it's used twice and make it a async await function, wrapping it in `useCallback` to be able to call it within synchronous useEffect.